scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -237,6 +254,39 @@ Function PrepareTestFromArtifacts
         (Get-Content $DeploymentWorkingFilePath).replace('<CR.Username>', $ContainerRegistryUsername) | Set-Content $DeploymentWorkingFilePath
         (Get-Content $DeploymentWorkingFilePath).replace('<CR.Password>', $ContainerRegistryPassword) | Set-Content $DeploymentWorkingFilePath
         (Get-Content $DeploymentWorkingFilePath).replace('-linux-', '-windows-') | Set-Content $DeploymentWorkingFilePath
+
+        If ($ProxyUri)
+        {
+            # Add/remove/edit JSON values *after* replacing all the '<>' placeholders because
```

Comment:
will it be easier if we have another copy of deployment file for proxy scenario?

Reply:
I don't think so, because we'd need to have several new files (possibly one for each of the seven tests), but with proxy info. That would mean **lots** of duplication in the deployment files.